colibre/scripts/wallclock_simulation_time.py

```diff
@@ -0,0 +1,63 @@
+"""
+Plots wallclock v.s. simulation time.
+"""
+
+import unyt
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+
+try:
+    plt.style.use("mnras.mplstyle")
+except:
+    pass
+
+from glob import glob
+from swiftpipeline.argumentparser import ScriptArgumentParser
+
+arguments = ScriptArgumentParser(
+    description="Creates a run performance plot: simulation time versus wall-clock time"
+)
+
+run_names = arguments.name_list
+run_directories = [f"{directory}" for directory in arguments.directory_list]
+snapshot_names = [f"{snapshot}" for snapshot in arguments.snapshot_list]
+output_path = arguments.output_directory
+
+plt.style.use(arguments.stylesheet_location)
+
+fig, ax = plt.subplots()
+
+for run_name, run_directory, snapshot_name in zip(run_names,
```

Comment:
Please format all of your scripts with `black`.

Reply:
Done